Solver/Dynamic/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ gridkit_add_library(solvers_dyn
   LINK_LIBRARIES
     PUBLIC SUNDIALS::nvecserial
     PUBLIC SUNDIALS::idas
+    PUBLIC SUNDIALS::sunlinsolklu
```

Comment:
If no sparse solver is available, I would default to a dense one. Linking to KLU should be conditional to SUNDIALS built with KLU support:
```cmake
if (GRIDKIT_ENABLE_SUNDIALS_SPARSE)
  target_link_libraries(solvers_dyn PUBLIC SUNDIALS::sunlinsolklu)
endif()
```

Reply:
To be handled in #9.